src/main/resources/application.properties

```diff
@@ -35,6 +35,9 @@ upload.static-path=classpath:static
 # URL that will serve static content
 upload.static-serve=http://localhost:3000/static
 
+# Frontend Url
+frontend.url = http://localhost:3000
```

Comment:
Really small detail but origin probably makes more sense than url. Perhaps it's even better to call it something like cors.allow-origin since it's the only use for the origin